Yoast/YoastSEO.js
# Improve the example with a lot of sass (not that sass)

## Summary

This PR can be summarized in the following changelog entry:

* _N/A_

## Relevant technical choices:

* Required `yoast-components` in the example to have access to awesome components.

## Test instructions

This PR can be tested by following these steps:

* Run `yarn start` in `/examples/webpack`.
* Inspect the dev tool to see if it works.
